keyboards/scrabblepad/keymaps/random/keymap.c

```diff
@@ -0,0 +1,75 @@
+#include "scrabblepad.h"
+#include <stdlib.h>
+
+static uint16_t keystroke;
```

Comment:
See comment regarding `random_keycode` return type.
```suggestion
static uint8_t keystroke;
```